src/__test__/redux/reducers/genes/markerGenesLoaded.test.js

```diff
@@ -25,6 +25,7 @@ describe('markerGenesLoaded', () => {
       payload: {
         genes,
         data,
+        plotUuid: 'interactiveHeatmap',
```

Comment:
change to `plotType`

Reply:
or actually, judging from file `src/components/data-exploration/heatmap/HeatmapPlot.jsx` it is `componentType`